src/objects/message.py

```diff
@@ -39,67 +45,39 @@ def __eq__(self, other):
         equality
         """
         try:
-            return self.sender == other.sender and self.message_type == other.message_type and self.content == other.content and self.timestamp == other.timestamp
+            return self.sender == other.sender and \
+                   self.message_type == other.message_type and \
+                   self.content == other.content and \
+                   self.timestamp == other.timestamp
         except AttributeError:
             return False
 
     # ------------ MEMBER ACCESS METHODS --------------
 
     @property
     def sender(self) -> int:
-        """
-        getter method for message sender
-        :return: integer message sender
-        """
         return self.__sender
 
-    @sender.setter
-    def sender(self, sender: int) -> None:
-        """
-        setter method for message sender
-        :param sender: integer sender
-        :return: None
-        """
-        self.__sender = sender
+    @property
+    def timestamp(self):
+        return self.__timestamp
 
     @property
     def content(self):
-        """
-        getter method for message content
-        :return: message contents
-        """
         return self.__content
 
+    @property
+    def message_type(self) -> MsgType:
+        return self.__message_type
+
+    @sender.setter
```

Comment:
Why `sender`, `content`, and `message_type` requires setter methods?

Reply:
doesn't. youre right.